lib/utils/AbstractQueryConfiguration.dart

```diff
@@ -31,23 +32,35 @@ abstract class AbstractQueryConfiguration {
 
   AbstractQueryConfiguration({
     this.language,
-    this.languages = const [],
+    this.languages,
     this.lc,
     this.cc,
     this.fields,
   }) {
     fields ??= [ProductField.ALL];
-    if ((language != null || lc != null) && languages.isNotEmpty) {
-      throw ArgumentError(
-          '[languages] cannot be used together with [language]/[lc]');
+    if (languages != null) {
+      if ((language != null || lc != null) && languages!.isNotEmpty) {
```

Comment:
The second check will always be true
![image](https://user-images.githubusercontent.com/3338378/133378602-fb5f245f-ae83-47f0-a331-d4a11dd3fcbe.png)

Reply:
It's language**s** and language, or have I missed something

Reply:
Oh, you didn't miss anything, it's me who have missed the fact that these variables are different :sweat_smile: 
Sorry!

Reply:
No problem, they look very similar though, what about renaming the local variable from languages to languageList